src/SDK/Trace/TracerProvider.php

```diff
@@ -66,7 +66,7 @@ public function getTracer(
         ?string $schemaUrl = null,
         iterable $attributes = []
     ): API\TracerInterface {
-        if ($this->tracerSharedState->hasShutdown() || SDK::isDisabled()) {
+        if ($this->tracerSharedState->hasShutdown() || Sdk::isDisabled()) {
```

Comment:
I think `Sdk::isDisabled()` should only affect `TracerProvider`s that are created from environment variables.

Reply:
Fair enough. I've left the one in MeterProvider for now, but noted that it also should be removed when we can create meter providers from env vars.